app/models.py

```diff
@@ -80,7 +73,13 @@ class UserGroup(db.Model):
 
 class Group(db.Model):
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(64), unique=True)
+    name = db.Column(db.String(64))
+    classCode = db.Column(db.String(6), nullable=True, unique=True)
+
+    def set_class_code(self):
+        lettersAndDigits = string.ascii_letters + string.digits
+        self.classCode = ''.join((random.choice(lettersAndDigits) for i in range(6)))
+        return self.classCode
```

Comment:
```suggestion
```
To remove since being done in `views.py`.

Reply:
Yea, removed after pushing and didn't push the updated one yet